gnosis/gp-ui
# [Explorer] Add local timezone on date fields

# Summary

Closes #613

Added local timezone to date fields using `date-fns` utils

![image](https://user-images.githubusercontent.com/11525018/134261060-685d1a38-2412-40b7-b8a8-0d63b0883d0d.png)

# To Test

1.  Open `home` page 
* Search for an order. i.e: `0x3bdceef7965b0f342aa37f7489007cd4ce2c9a07b485571351f133943d23d4cab6bad41ae76a11d10f7b0e664c5007b908bc77c961144d83`
* You will see local timezone next to the field dates (Submission|Expiration time)
